pkgs/applications/blockchains/snarkvm/default.nix

```
@@ -0,0 +1,48 @@
+{
+  fetchFromGitHub,
+  lib,
+  rustPlatform,
+  pkg-config,
+  openssl,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "snarkVM";
+  version = "1.2.1";
+
+  src = fetchFromGitHub {
+    owner = "ProvableHQ";
+    repo = pname;
```

`repo = pname` is an antipattern as the two values are not necessarily related.
```suggestion
    repo = "snarkVM";
```